index.html

```diff
@@ -0,0 +1,138 @@
+                    class="big2">Games?</div>
+            </h1>
+            <div class="strong">
+                <h2><a href="">Workshop+Talk w/ Three Famous Game Designers</a></h2>
+                <h2><a href="">Dec 20th, 10.30 AM EST (Zoom)</a></h2>
+                <h2><a href="">These events are free and open to the public.
+                        No prior game design experience required.</a></h2>
+            </div>
+
+            <h4>Game Designers craft the core mechanics of a game.
+                This means that they are responsible for deciding how games work when the player starts playing
+                them.
+                Most
+                importantly, game designers create rules.
+                These rules govern how games work.
+                In this workshop, participants will be introduced to what problems and rules the game designers will
+                be
+                facing
+                when
+                desiging games through speaks' life experience.
+            </h4>
+        </div>
+        <ul class="u-list-reset speaker-group">
+            <li class="speakers">
+                <div class="author-company">
+                    <img src="https://cdn.glitch.com/46be344e-bd86-488e-bf75-ab4ed5228fee%2Falex_jaffe-2c0a51ae.jpg?v=1608230332266"
+                        alt="Alex Jaffe" class="author-img">
+                    <div class="speaker-title">
+                        <h4>Alex Jaffe</h4>
+                        <h5>A game Designer in R&D at Riot Games.</h5>
+                        <h4> <a href="https://www.gdcvault.com/play/1025756/Cursed-Problems-in-Game">Cursed Problems in
+                                Game Design</a></h4>
+                    </div>
+                </div>
+                <footer>
+                    <p> Game designers love solving problems, perfecting games until they
+                        sing. But sometimes you hit a design problem so thorny that you
+                        just can't find a solution, and maybe never will. This may be a
+                        "cursed problem": a problem with some inherent contradiction of
+                        goals. Consider the kingmaker problem. A designer wants to
+                        emphasize certain player skills, but also wants free-for-all PvP,
+                        which incentivizes politics in favor of those skills. Cursed problems
+                        like these haunt designers across game and genre. And somehow
+                        all designers have to deal with them.In this talk, Alex Jaffe (Riot
+                        Games) will present a theory of cursed problems. He'll show case
+                        studies of decades-old problems, and he'll explore the four design
+                        paradigms that have helped developers face these problems and
+                        survive.
+                    </p>
+                </footer>
```

Comment:
If there is consistently only one item inside this `footer`, you can eliminate the `footer` element and just style the `p` element directly.